assets/html/scss/documenter-dark.scss

```diff
@@ -122,4 +122,13 @@ html.theme--#{$themename} {
   .hljs-subst {
     color: #f8f8f2;
   }
+  
+  .search-result-link:hover .property-search-result-badge, .search-result-link:focus .property-search-result-badge {
+    color: $search-333 !important;
+    background-color: $search-f1f5f9 !important;
+  }
+  
+  .search-result-title {
+    color: $search-whitesmoke !important;
+  }
```

Comment:
I think we should just be able to do this by overriding the variables in `documenter-dark.scss`, not the whole selectors?